### PR TITLE
improve benchmark macro

### PR DESCRIPTION
**Pull request by zqhxuyuan**
closes: https://github.com/open-web3-stack/open-runtime-module-library/issues/690

### CODE REVIEW

File: benchmarking/src/lib.rs [hunk 1]
```diff
@@ -1358,3 +1372,53 @@ macro_rules! list_benchmark {
...
+    ( $([ $names:path, $( $locations:tt )* ])* ) => {
+		/// Calls `list_benchmark` with all configs from `define_benchmarks`
+		/// and passes the first two parameters on.
+		///
+		/// Use as:
+		/// ```ignore
+		/// list_benchmarks!(list, extra);
+		/// ```
+		#[macro_export]
+        macro_rules! list_benchmarks {
```
Comment (shaunxw) [inline benchmarking/src/lib.rs:1404]: indent level seems wrong

File: benchmarking/src/lib.rs [hunk 2]
```diff
@@ -1358,3 +1372,53 @@ macro_rules! list_benchmark {
...
+
+		/// Calls `add_benchmark` with all configs from `define_benchmarks`
+		/// and passes the first two parameters on.
+		///
+		/// Use as:
+		/// ```ignore
+		/// add_benchmarks!(params, batches);
+		/// ```
+		#[macro_export]
+        macro_rules! add_benchmarks {
```
Comment (shaunxw) [inline benchmarking/src/lib.rs:1418]: same here

**Review by shaunxw — COMMENTED** [review]
can you confirm it works in acala/karura/mandala runtime before merging?

File: benchmarking/src/lib.rs (hunk 1 above)
Comment (shaunxw) [reply benchmarking/src/lib.rs:1404]: hmmm, looks right again in PR conversation page, probably a github front end bug

Comment (xlc) [reply benchmarking/src/lib.rs:1404]: it is tab vs space thing

**Review by shaunxw — APPROVED** [review]

**Pull request merged**